amppkg.example.toml

```diff
@@ -203,57 +208,62 @@ ForwardedRequestHeaders = []
 # setup to do automatic cert renewals and the rest of the instances will just be configured to reload the fresh
 # certificate from disk when their in-memory copies expire.  This also implies that the cert paths configured above
 # in 'CertFile' and 'NewCertFile' are located on a shared filesystem accessible by all AMP packager instances.
+#
+# For the full ACME spec, see:
+# 	https://tools.ietf.org/html/draft-ietf-acme-acme-02
+# 	https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html
+# TODO(banaag): consider renaming ACMEConfig to ACME
 # [ACMEConfig]
-  # [ACMEConfig.Production]
   # This config will be used if 'autorenewcert' is turned on and 'development' is turned off.
-    # This is the ACME discovery URL that is used for ACME http requests to the Certificate authority that
+  # If the flags above are on but we don't have an entry here, AMP Packager will not start.
+  # [ACMEConfig.Production]
+    # This is the ACME discovery URL that is used for ACME http requests to the Certificate Authority that
     # doles out the certificates.
     # Currently, the only CA that supports automatic signed exchange cert renewals is Digicert:
     # https://docs.digicert.com/certificate-tools/acme-user-guide/acme-directory-urls-signed-http-exchange-certificates/
-    #
-    # DiscoURL = "production-acme.discovery.url"
-    #
+
```

Comment:
Remove this blank line, so each comment is grouped with its field. So the format is like:

```
  # Comment explaining Field. Blah blah lorem
  # ipsum blah.
  # Field = "foo"

  # Comment explaining OtherField. Oh wow so
  # interesting.
  # OtherField = true
```